### PR TITLE
Updated migration guide as more methods now throw exceptions.

### PR DESCRIPTION
**Pull request by ahus1**
Related to keycloak/keycloak#12556

### CODE REVIEW

**Comment by ahus1** [comment]
Hi @mhajas - as you've been involved in the original issue, ask you to review this documentation.
Please let me know if this works for you, or you'd want @hmlnarik to have a look instead.

**Review by mhajas — DISMISSED** [review]
Thanks @ahus1. LGTM

**Review by hmlnarik — DISMISSED** [review]

File: upgrading/topics/keycloak/changes-19_0_0.adoc [hunk 1]
```diff
@@ -49,19 +49,23 @@ Several methods have been deprecated in `KeycloakSession` and will be removed in
 This situaton may be confusing for the developer who has to understand the exact meaning of each method, and depends on current store layout.
 The new store does not distinguish federated from local storage.
 
-For both reasons, only the `users()` methods will be kept in `KeycloakSession` and the rest of methods are currently deprecated, return the same object as `users()`, and will eventually be removed.
-The same pattern of deprecation applies to methods all other object areas, e.g. `clients()` or `groups()`.
+For those reasons, only the `users()` method will be kept in `KeycloakSession`, and should replace all other calls listed above.
+The rest of the methods are deprecated, and will eventually be removed.
+The same pattern of deprecation applies to methods of other object areas, e.g. `clients()` or `groups()`.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-19_0_0.adoc]:
```suggestion
The same pattern of deprecation applies to methods of other object areas, such as `clients()` or `groups()`.
```

File: upgrading/topics/keycloak/changes-19_0_0.adoc [hunk 2]
```diff
@@ -49,19 +49,23 @@ Several methods have been deprecated in `KeycloakSession` and will be removed in
...
+All methods `++*StorageManager()++` and `++*LocalStorage++()` now throw an exception when being called, as there is no direct replacement in the new store.
+The next section describes how to migrate those calls to the new API, or use the legacy API while using the old store.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-19_0_0.adoc]:
```suggestion
The next section describes how to migrate those calls to the new API or use the legacy API while using the old store.
```

File: upgrading/topics/keycloak/changes-19_0_0.adoc [hunk 3]
```diff
@@ -49,19 +49,23 @@ Several methods have been deprecated in `KeycloakSession` and will be removed in
 This situaton may be confusing for the developer who has to understand the exact meaning of each method, and depends on current store layout.
 The new store does not distinguish federated from local storage.
 
-For both reasons, only the `users()` methods will be kept in `KeycloakSession` and the rest of methods are currently deprecated, return the same object as `users()`, and will eventually be removed.
-The same pattern of deprecation applies to methods all other object areas, e.g. `clients()` or `groups()`.
+For those reasons, only the `users()` method will be kept in `KeycloakSession`, and should replace all other calls listed above.
+The rest of the methods are deprecated, and will eventually be removed.
+The same pattern of deprecation applies to methods of other object areas, e.g. `clients()` or `groups()`.
+All methods `++*StorageManager()++` and `++*LocalStorage++()` now throw an exception when being called, as there is no direct replacement in the new store.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-19_0_0.adoc]: Should it be "All methods of..." ?

File: upgrading/topics/keycloak/changes-19_0_0.adoc [hunk 4]
```diff
@@ -49,19 +49,23 @@ Several methods have been deprecated in `KeycloakSession` and will be removed in
...
+The next section describes how to migrate those calls to the new API, or use the legacy API while using the old store.
 
 The deprecated methods in KeycloakSession will be removed in a future release.
-The `keycloak-model-legacy-*` modules will be available for a longer time, but they will eventually be removed.
+The `keycloak-model-legacy-*` modules will be available for a longer time, and will eventually be removed.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-19_0_0.adoc]:
```suggestion
The `keycloak-model-legacy-*` modules will be available for a longer time and will eventually be removed.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, @ahus1. Just a few suggestions.

As always, we avoid latin.  We use for example instead of e.g.

File: upgrading/topics/keycloak/changes-19_0_0.adoc (hunk 3 above)
Comment (ahus1) [reply upgrading/topics/keycloak/changes-19_0_0.adoc]: I'll make this "All methods ending in ...", as the "*" is  a placeholder.

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good @ahus1

**Review by andymunro — APPROVED** [review]

**Comment by ahus1** [comment]
Thank you for that review - now I'll start looking for a maintainer. I also put one more Latin word to my auto-correction...

**Pull request merged**